content/en/docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Try Falco on Kubernetes
+description: Try Falco on Kubernetes
+slug: falco-kubernetes-quickstart
+aliases: [/falco-kubernetes-quickstart]
+weight: 25
+---
+
+## Introduction
+Falco is designed to provide real-time notification of suspicious behavior. By default, it can send these alerts to stdout, stdrerr, an HTTPs endpoint, and a gRPC endpoint. However, in most production scenarios, you will want to send the output to some other system so you can take further action. Perhaps you would forward notification to Elastic or an SIEM. 
```

Comment:
```suggestion
Falco is designed to provide real-time notification of suspicious behavior. By default, it can send these alerts to stdout, stdrerr, an HTTPs endpoint, and a gRPC endpoint. However, in most production scenarios, you will want to send the output to some other system so you can take further action.
```